app/helpers/operatingsystems_helper.rb

```diff
@@ -83,7 +83,7 @@ def os_default_templates_for_form(os)
     if os.os_default_templates.any?(&:new_record?)
       os.os_default_templates.sort_by { |o| o.template_kind.name.downcase }
     else
-      os.os_default_templates.joins(:template_kind).order('LOWER(template_kinds.name)')
+      os.os_default_templates.joins(:template_kind).order(Arel.sql('LOWER(template_kinds.name)'))
```

Comment:
```suggestion
      os.os_default_templates.joins(:template_kind).order(TemplateKind.arel_table[:name].lower)
```

Untested, but this can be expressed with arel directly.

Reply:
indeed :+1: